packages/app/src/cli/constants.ts

```diff
@@ -36,14 +36,14 @@ export const blocks = {
   },
 } as const
 
-export const defaultFunctionsFlavors: {name: string; value: ExtensionFlavor}[] = [
+export const defaultFunctionsFlavors: ExtensionFlavor[] = [
   {name: 'JavaScript (developer preview)', value: 'vanilla-js'},
   {name: 'TypeScript (developer preview)', value: 'typescript'},
   {name: 'Rust', value: 'rust'},
   {name: 'Wasm', value: 'wasm'},
 ]
```

Comment:
The Partners PR has been merged, we can remove all references to local function specifications right?